src/binance/bnbInfo.ts

```diff
@@ -6,11 +6,11 @@ import { BinanceSettings } from './bnbTypes'
 
 const otherSettings: BinanceSettings = {
   binanceApiServers: [
-    'https://dex.binance.org',
```

Comment:
I think this endpoint is still serving requests and shouldn't be removed.

Reply:
I'll add back. I think it was a mistaken removal.